test/test_events.py

```diff
@@ -179,10 +179,12 @@ def test_seminar_events_typical(uqcsbot: MockUQCSBot):
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
     expected = ("Events in the *next _2_ weeks*\r\n"
-                "*MAY 28 12:00 - 13:00* - `<https://www.itee.uq.edu.au/introduction"
+                "*TUE MAY 28 12:00 - 13:00* - "
+                "`<https://www.itee.uq.edu.au/introduction"
```

Comment:
Because the message continues on the same line, this line of code should begin with a `+`.
Furthermore, I would prefer the joining characters to be on the new line, so the line should be  ```+ " - `<https://www.itee.uq.edu.au/introduction"``` (and remove the ` - ` from the previous line).